docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md

```diff
@@ -11,7 +11,7 @@ Limit the throughput to a specific number of elements per time unit, or a specif
 
 ## Description
 
-Limit the throughput to a specific number of elements per time unit, or a specific total cost per time unit, where
+Limit the throughput to a specific number of elements per time unit, or a specific total cost per-time unit, where
```

Comment:
I think it really is "cost per time unit", no idea what "cost per-time unit" would mean

Reply:
```suggestion
Limit the throughput to a specific number of elements per time unit, or a specific total cost per time unit, where
```